oracle/src/oracleimporter.h

```diff
@@ -2,6 +2,7 @@
 #define ORACLEIMPORTER_H
 
 #include <QMap>
+#include <QVariant>
```

Comment:
Before, ``` #include <carddatabase.h>``` would get in some includes for Qt objects that included <QVariant> themselves. With this PR a lot of includes have been removed from carddatabase.h, and i had to add this to fix compilation.